core/src/main/java/apoc/trigger/TriggerHandler.java

```diff
@@ -35,6 +35,7 @@
 public class TriggerHandler extends LifecycleAdapter implements TransactionEventListener<Void> {
 
     private enum Phase {before, after, rollback, afterAsync}
+    private static final Map<String, Object> TRIGGER_META = Map.of("trigger", true);
```

Comment:
`Map.of("apoc.trigger", true)` would be better in my opinion. Just in case people start seeing things in their transactions they don't know where they came from, that's more self explanatory than just trigger